docs/reference/using/cli/gel_extension/list-available.rst

```diff
@@ -0,0 +1,27 @@
+.. _ref_cli_gel_extension_list_available:
+
+============================
+gel extension list-available
+============================
+
+List available extensions.
+
+.. cli:synopsis::
+
+  gel extension list-available [<options>]
```

Comment:
Unrelated and nitpick, but is there a plan for a shortcut for this? `list-available` is long to type 😅

Reply:
Eh, hopefully this something you need to do often enough for it to be annoying 😅 Maybe one day we can have an alias feature like git.